package.json

```diff
@@ -198,7 +198,7 @@
     "ember-sinon": "^5.0.0",
     "ember-sinon-qunit": "^6.0.0",
     "ember-sortable": "^5.0.0",
-    "ember-source": "~3.28.8",
+    "ember-source": "~3.28.12",
```

Comment:
Is this the only thing that needs to change if we're bumping versions? Or are there other ember dependencies that also need to be updated?

Also, is this necessary for your PR, or would it make sense to do this as a separate PR so we can explore the above questions?